homeassistant/components/aprilaire/sensor.py

```diff
@@ -0,0 +1,254 @@
+        status_sensor_available_value=0,
+        status_sensor_exists_values=[0, 1, 2],
+        value_key=Attribute.INDOOR_TEMPERATURE_CONTROLLING_SENSOR_VALUE,
+        value_fn=lambda sensor, value: convert_temperature_if_needed(
+            sensor.unit_of_measurement, value
+        ),
+    ),
+    AprilaireSensorDescription(
+        key="outdoor_temperature_controlling_sensor",
+        translation_key="outdoor_temperature_controlling_sensor",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        status_key=Attribute.OUTDOOR_TEMPERATURE_CONTROLLING_SENSOR_STATUS,
+        status_sensor_available_value=0,
+        status_sensor_exists_values=[0, 1, 2],
+        value_key=Attribute.OUTDOOR_TEMPERATURE_CONTROLLING_SENSOR_VALUE,
+        value_fn=lambda sensor, value: convert_temperature_if_needed(
+            sensor.unit_of_measurement, value
+        ),
+    ),
+    AprilaireSensorDescription(
+        key="dehumidification_status",
+        translation_key="dehumidification_status",
+        device_class=None,
+        state_class=None,
+        status_key=Attribute.DEHUMIDIFICATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.DEHUMIDIFICATION_STATUS,
+        value_fn=lambda _, value: DEHUMIDIFICATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="humidification_status",
+        translation_key="humidification_status",
+        device_class=None,
+        state_class=None,
+        status_key=Attribute.HUMIDIFICATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1, 2],
+        value_key=Attribute.HUMIDIFICATION_STATUS,
+        value_fn=lambda _, value: HUMIDIFICATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="ventilation_status",
+        translation_key="ventilation_status",
+        status_key=Attribute.VENTILATION_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.VENTILATION_STATUS,
+        value_fn=lambda _, value: VENTILATION_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="air_cleaning_status",
+        translation_key="air_cleaning_status",
+        status_key=Attribute.AIR_CLEANING_AVAILABLE,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[1],
+        value_key=Attribute.AIR_CLEANING_STATUS,
+        value_fn=lambda _, value: AIR_CLEANING_STATUS_MAP.get(value),
+    ),
+    AprilaireSensorDescription(
+        key="fan_status",
+        translation_key="fan_status",
+        status_key=None,
+        status_sensor_available_value=None,
+        status_sensor_exists_values=[],
+        value_key=Attribute.FAN_STATUS,
+        value_fn=lambda _, value: FAN_STATUS_MAP.get(value),
```

Comment:
these status sensors look to be returning a default set of names. You can make these an enum sensor. The added value of this is that you have to pass in a list of possible values (via `options`) and when users want to automate with it for example, they can see what possible values the entity returns.

This does require that the values returned by the sensor are in `snake_case` and then you can add state translations in the strings.json.